week4-ts/src/components/ResultSection.tsx

```diff
@@ -0,0 +1,31 @@
+import { Store } from "../types";
+import Skeleton from "../components/Skeleton";
+import StoreCard from "./StoreCard";
+
+interface ResultSectionProps {
+  storeInfo: Store[];
+  isCheck: boolean;
+  isLoading: boolean;
+}
+
+export default function ResultSection({
+  storeInfo,
+  isCheck,
+  isLoading,
+}: ResultSectionProps) {
+  return (
+    <>
```

Comment:
컨벤션 차이이지만, 아래처럼 선언하기도 합니다! 가독성이 좋다는 의견이 있던 것 같아요,, 는 주함

```suggestion
export default function ResultSection(props: ResultSectionProps) {
  const {storeInfo, isCheck, isLoading} = props

  return (
    <>
```

Reply:
오..훨씬 보기 편하네요 감사합니다!